specs/eventing/control-plane.md

```diff
@@ -0,0 +1,985 @@
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>physicalSubscription</code></td>
+    <td><a href="#physicalsubscriptionstatus">PhysicalSubscriptionStatus</a></td>
+    <td>The fully resolved values for <code>spec</code> endpoint references.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### Addressable v1
+
+Note that the Addressable interface is a partial schema -- any resource which
+includes these fields may be referenced using a `duckv1.Destination`.
+
+#### Metadata:
+
+Standard Kubernetes
+[metav1.ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#objectmeta-v1-meta)
+resource.
+
+#### Spec:
+
+There are no `spec` requirements for Addressable.
+
+#### Status
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>address</code></td>
+    <td><a href="#duckv1addressable">duckv1.Addressable</a></td>
+    <td>Address used to deliver events to the Broker.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+## Detailed SubResource Objects
+
+### duckv1.Addressable
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>url</code></td>
+    <td>URL (string)</td>
+    <td>Address used to deliver events to the Addressable.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### duckv1.Destination
+
+Destination is used to indicate the destination for event delivery. A
+Destination eventually resolves the supplied information to a URL by resolving
+`uri` relative to the address of `ref` (if provided). `ref` may be an
+[Addressable](#duckv1-addressable) object or a `v1/Service` Kubernetes service.
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>ref</code></td>
+    <td><a href="#duckv1kreference">duckv1.KReference</a><br/>(Optional)</td>
+    <td>An ObjectReference to an Addressable reference to deliver events to.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>uri</code></td>
+    <td>URL (string)<br/>(Optional)</td>
+    <td>A resolved URL to deliver events to.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### duckv1.SubscriberSpec
+
+SubscriberSpec represents an automatically-populated extraction of information
+from a [Subscription](#subscription).
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>uid</code></td>
```

Comment:
If I'm following, I think this may need to REQUIRED to be supported by implementations. If people write code to change the list of Subscriptions and they are counting on this UID being there so they can properly identify which one they want to touch, then they will have interop issues if we don't require everyone to support it.